server.go

```diff
@@ -842,8 +886,9 @@ func (s *Server) HandleGetPayload(w http.ResponseWriter, r *http.Request) {
 	marshalUnmarshalSpan.End()
 	w.Header().Set(common.HeaderEthConsensusVersion, payloadResponse.Version.String())
 	s.respondOKWithContextSSZMarshalled(getPayloadCtx, span, method, w, outByte, &log, s.tracer)
```

Comment:
you need to change respondOKWithContextSSZMarshalled aswell if you are changing respondOK.